oauth2_client/models.py

```diff
@@ -136,7 +135,6 @@ class AccessToken(models.Model):
         null=True
     )
 
-    # Scope is a series of space delimited strings, eg. "read write"
```

Comment:
This information might be useful to casual readers of _fetcher.py_ unless the representation is hidden under some abstraction.